src/Core/MerchandiseReturn/Configuration/MerchandiseReturnOptionsConfiguration.php

```diff
@@ -26,36 +26,32 @@
 
 namespace PrestaShop\PrestaShop\Core\MerchandiseReturn\Configuration;
 
-use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
-use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Configuration\AbstractMultistoreConfiguration;
+use PrestaShopBundle\Form\Admin\Sell\CustomerService\MerchandiseReturn\MerchandiseReturnOptionsType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * Provides data configuration for merchandise returns options form
  */
-final class MerchandiseReturnOptionsConfiguration implements DataConfigurationInterface
+class MerchandiseReturnOptionsConfiguration extends AbstractMultistoreConfiguration
 {
-    /**
-     * @var ConfigurationInterface
-     */
-    private $configuration;
-
-    /**
-     * @param ConfigurationInterface $configuration
-     */
-    public function __construct(ConfigurationInterface $configuration)
-    {
-        $this->configuration = $configuration;
-    }
+    private const CONFIGURATION_FIELDS = [
+        MerchandiseReturnOptionsType::FIELD_ENABLE_ORDER_RETURN,
+        MerchandiseReturnOptionsType::FIELD_ORDER_RETURN_PERIOD_IN_DAYS,
+        MerchandiseReturnOptionsType::FIELD_ORDER_RETURN_PREFIX,
+    ];
```

Comment:
Architecture bad practice: Core components should never use Bundle constants. The Bundle should depend on the Core, not the other way around.
Instead public constants should be defined here in Core, and then used by the form type